db/sequent_schema_tables.sql

```diff
@@ -1,6 +1,6 @@
-CREATE TABLE command_types (id SMALLINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, type text UNIQUE NOT NULL);
-CREATE TABLE aggregate_types (id SMALLINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, type text UNIQUE NOT NULL);
-CREATE TABLE event_types (id SMALLINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, type text UNIQUE NOT NULL);
+CREATE TABLE command_types (id SMALLINT PRIMARY KEY, type text UNIQUE NOT NULL);
```

Comment:
Can we have something like `migrate_metadata_tables` ? Or use AR migrations for this.

Reply:
That would be nice at some point, for now I added a quick fix to the new pgpsql file that drops these IDENTITY declarations if they exist.